src/models/brackets.rs

```diff
@@ -437,33 +439,26 @@ impl Bracket {
                 Outcome::P1Win => (2, 0),
                 Outcome::P2Win => (0, 2),
             };
-            let p1_i = info
+            let p1_i_b = info
```

Comment:
Curious, what does the `_b` represent?

Reply:
builder - you can see above that the values in `info` are of type `PlayerInfoBuilder`, not `PlayerInfo`, so the variable was named  bit confusingly

Reply:
Aah